frontend-react/src/components/AuthElement.tsx

```diff
@@ -22,7 +22,7 @@ export const AuthElement = ({
     const { oktaToken, activeMembership } = useSessionContext();
     // A way to check if this is a logged-in user refreshing the app
     const tokenAvailable = useMemo(
-        () => getStoredOktaToken() === undefined || getStoredOktaToken() === "",
+        () => getStoredOktaToken() !== undefined && getStoredOktaToken() !== "",
```

Comment:
why isn't null included here?

Reply:
I don't believe it can return `null`. The actual sessionStorage API gives undefined if no key/value pair is matched, and we have a layer atop that and it sometimes, for some reason not all the time (noted by it's possibly undefined return value) makes it an empty string inside of the `getStoredOktaToken` function, so either response of a token missing should be covered in this. 
Is there a benefit of using `null` vs `undefined` for checks like these? It seems most JS APIs and libraries lean into undefined, but I grew up with java so null makes sense...is `undefined === null`?

Reply:
good question, because javascript is very whack on this point. apologies if I'm saying stuff you already know:
```
undefined !== null
undefined == null

undefined !== ""
undefined == ""
```
`undefined`, `null` and `""` (and `NaN` and `0` and `false` for that matter, but no need to drag them into this) are all FALSEY, which means they are all `!<value> ===  false, !!<value> === true`

In a case like this, based on what you're saying, I think we're safe to say 
```suggestion
        () => !!getStoredOktaToken(),
```